integration/inspektor-gadget/integration_test.go

```diff
@@ -1185,10 +1186,24 @@ func TestSnisnoop(t *testing.T) {
 	t.Parallel()
 
 	snisnoopCmd := &Command{
-		Name:           "StartSnisnoopGadget",
-		Cmd:            fmt.Sprintf("$KUBECTL_GADGET trace sni -n %s", ns),
-		ExpectedRegexp: fmt.Sprintf(`%s\s+test-pod\s+kinvolk.io`, ns),
-		StartAndStop:   true,
+		Name:         "StartSnisnoopGadget",
+		Cmd:          fmt.Sprintf("$KUBECTL_GADGET trace sni -n %s -o json", ns),
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntry := &sniTypes.Event{
+				Event: BuildBaseEvent(ns),
+				Name:  "kinvolk.io",
```

Comment:
I would suggest using `inspektor-gadget.io` instead.

Reply:
Sounds better. I have updated the PR.